examples/.bazelversion

```diff
@@ -0,0 +1 @@
+7.4.1
```

Comment:
Please make this a symlink to the root repo' file.

Reply:
Done, but neither the symlink nor the full file result in the expect behavior. I still see bazel 8 being used: https://github.com/mvukov/rules_ros/actions/runs/12339442266/job/34435694166?pr=88#step:5:9